config/webpack-config.js

```diff
@@ -36,3 +36,40 @@ module.exports = {
     new BannerFooterPlugin( '{% raw %}', '{% endraw %}', { raw: true } )
   ]
 };
+
+var ieConf = {
+  entry: paths.unprocessed + '/js/ie/common.ie.js',
+  output: {
+    filename: 'common.ie.js'
+  },
+  plugins: [
+    new webpack.optimize.UglifyJsPlugin( {
+      compress: { warnings: false }
+    } )
+  ]
+};
+
+var atomicConf = {
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/pull/1796/commits/49335e3b7094033c1037c11fda4f773feb98ba0e